Application/EdFi.Ods.Tests/EdFi.Ods.Security/Authorization/EdFiAuthorizationProviderTests.cs

```diff
@@ -447,7 +447,7 @@ public abstract class When_authorizing_a_request_affected_by_authorization_strat
                 // Issue Resource claim with action
                 Claim[] claims =
                 {
-                    JsonClaimHelper.CreateClaim(resourceClaimUri, new EdFiResourceClaimValue(actionUri, authorizationStrategyNameOverride))
+                    JsonClaimHelper.CreateClaim(resourceClaimUri, new EdFiResourceClaimValue(actionUri, new List<string> { authorizationStrategyNameOverride }))
```

Comment:
Convert to an array rather than creating lists.

Reply:
I noticed that you have updated it